azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -18,16 +18,16 @@
```

Comment:
As per discussion, we should consider using a twin object with some kind of reported_properties delta